### ISSUE TITLE
YOLOv11n

### ISSUE THREAD

**Issue by OmarKhater98** [ultralytics/ultralytics#17120]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How I can train Yolov11n from scratch, not fine-tuning !!(with random weights not pre-trained weights)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @OmarKhater98, thank you for reaching out and your interest in Ultralytics 🚀! This is an automated response, but rest assured, an Ultralytics engineer will assist you soon.

We recommend visiting the [Docs](https://docs.ultralytics.com) for a detailed guide on training strategies, including examples for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/).

If you're looking to train a model from scratch with random weights (instead of fine-tuning), you'll want to adjust the `--weights` parameter to `''` during training. This will ensure that no pre-trained weights are used. If this doesn't answer your inquiry, or if further clarification is needed, please provide additional details, such as the command you’re trying to execute or any errors you encounter.

For bug-related issues, make sure to provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and solve any problems 🌟.

Join our vibrant Ultralytics community for more interactive help. For real-time chat support, head over to [Discord](https://ultralytics.com/discord) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com), or engage with us on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Please ensure your `ultralytics` package is up-to-date with all necessary [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in any of the following up-to-date, verified environments, with necessary dependencies ([CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), [PyTorch](https://pytorch.org/)) pre-installed:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This badge indicates the current status of our CI tests. If it's green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing, ensuring the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/).

**Comment by Y-T-G** [comment]
Use `.yaml` instead of `.pt`.

```python
from ultralytics import YOLO, ASSETS

model = YOLO("yolo11n.yaml")
```

**Comment by OmarKhater98** [comment]
@Y-T-G at which part I will use ASSETS?

**Comment by Y-T-G** [comment]
You can ignore that

**Comment by OmarKhater98** [comment]
Can you please share the sequence for the code if possible honestly, I tried to implement it and still face some issues @Y-T-G

**Comment by Y-T-G** [comment]
The code for training is the same as in the docs.
https://docs.ultralytics.com/modes/train/

The only difference is you use `.yaml` instead of `.pt`.

```python
from ultralytics import YOLO

model = YOLO("yolo11n.yaml")
results = model.train(data="coco128.yaml", epochs=10) 
```
